docs/assets/js/main.js

```diff
@@ -75,9 +75,45 @@ function injectFooterCopywrite() {
     footer.innerHTML = `&copy;2008-${new Date().getFullYear()} Expensify, Inc.`;
 }
 
+// Count property of --scroll-y variable
+window.addEventListener('scroll', () => {
+    document.documentElement.style.setProperty('--scroll-y', `${window.scrollY}px`);
+});
```

Comment:
Let's move this listened inside of `DOMContentLoaded` event https://github.com/Expensify/App/blob/01fc9285e5eeff2ac79173d799fad75f9d624d99/docs/assets/js/main.js#L78